open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/ProcessContextHandler.java

```diff
@@ -103,8 +107,8 @@ public ProcessContextHandler(String serviceName,
                                                                                                                PropertyServerException,
                                                                                                                UserNotAuthorizedException {
 
-        List<EntityDetail> entitiesTillLastRelationship = getEntitiesLinkedWithProcessPort(userId, entityDetail);
-        if(!entitiesTillLastRelationship.isEmpty()){
+        boolean entitiesTillLastRelationshipExist = getEntitiesLinkedWithProcessPort(userId, entityDetail);
```

Comment:
Might be better if you rename getEntitiesLinkedWithProcessPort to hasEntititesLinkedWithProcessPort